### DISCUSSION TITLE
Time Series Clustering via DTW

### DISCUSSION THREAD

**Discussion by sisupalan** [pycaret/pycaret#2356] [Ideas]
Love the pycaret ecosystem! It would be wonderful if we could have a DTW clustering module within time series! Looking forward to further pycaret updates!

**Comment by ngupta23** [comment]
@sisupalan We also would like to add other time-series tasks to the ecosystem. The time series ecosystem is based on `sktime` and `sktime` itself supports clustering. But currently, we have limited development resources so we are focusing on only the forecasting aspect. 

However, if someone familiar with time series clustering would like to take the lead on the development of this module in pycaret, we would be happy to have a discussion and discuss the next steps for the development.
